samples/Calling/src/app/App.tsx

```diff
@@ -64,8 +64,8 @@ const App = (): JSX.Element => {
     return <UnsupportedBrowserPage />;
   }
 
-  if (isMobileSession() || isSmallScreen()) {
-    console.log('ACS Calling sample: This is experimental behaviour');
+  if (isMobileSession() || isLandscape()) {
```

Comment:
and instead of or here
```suggestion
  if (isMobileSession() && isLandscape()) {
```